### PR TITLE
Reduce unneeded code

### PR DESCRIPTION
**Pull request by MichWouters**

### CODE REVIEW

File: DateTimeConverter.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                return $"{remainingMinutes} minutes";
+            }
+        }
+
+
+    }
+
+    public enum TimeInterval
+    {
+        Seconds,
```
Comment (MichielAltAccount) [inline DateTimeConverter.cs]: We should not have multiple classes / enums in one file. Please move this into its own file.

File: DistanceConverter.cs [hunk 2]
```diff
@@ -10,16 +10,15 @@ public double MilesToKilometers(double valueToConvert, int precision = 2)
             converted = Math.Round(converted, precision);
 
             Console.WriteLine($"Result: {converted}");
```
Comment (MichielAltAccount) [inline DistanceConverter.cs:12]: Do we need the logging?

**Review by MichielAltAccount — CHANGES_REQUESTED** [review]
Please move the enum in a seperate file

File: DateTimeConverter.cs (hunk 1 above)
Comment (MichWouters) [reply DateTimeConverter.cs]: Moved into a separate file.

**Review by MichielAltAccount — APPROVED** [review]

**Pull request merged**